status-go-version.json

```diff
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.179.0",
-    "commit-sha1": "0db27a8be6b482c3d4687909e156e38654911f88",
-    "src-sha256": "0qcdf90sz0y0486dsgjindanavilkh9czzijlk8r0zz867n5b896"
+    "version": "bump-go-to-1-20",
```

Comment:
Shouldn't this be a tagged release on go side too?

Reply:
it should be, atm its just to see if builds pass or fail.
I'm awaiting approvals on status-go side.
I'll mark this PR as draft for now :)

Reply:
got the approvals, I'll push a tag on status-go side once QA approves this PR.
Thats the process -> https://github.com/status-im/status-mobile/blob/develop/doc/merging-pr-process.md#status-go-changes

Reply:
tag pushed, status-go PR is merged.